wagtail/contrib/modeladmin/helpers/search.py

```diff
@@ -28,39 +27,39 @@ def show_search_form(self):
 
 
 class DjangoORMSearchHandler(BaseSearchHandler):
-    def search_queryset(self, queryset, search_term, **extra_search_kwargs):
+    def search_queryset(self, queryset, search_term, distinct_applied, **extra_search_kwargs):
         if not search_term or not self.search_fields:
-            return queryset, False
+            return queryset
 
         if extra_search_kwargs:
             queryset = queryset.filter(**extra_search_kwargs)
```

Comment:
@seb-b What's the thinking behind this bit? I'm cautious of blindly interpreting `kwargs` as additional filter arguments here (i don't think the design is clear enough). Also, I'm not sure this is necessary, since `queryset` can be modified freely before passing to the method (if necessary).

Reply:
The idea was to be able to use the `search_handler_extra_search_kwargs` option to filter the search results, which I don't believe is possible at the moment without overriding the index class. So it should probably be moved to the top of the method def now that I think about it. 
Either way I'm not particulary attached to it, it was just a possible convenience for some devs, happy to remove.

Reply:
Okay. I think developers can already override the `ModelAdmin.get_queryset()`, which receives the current `request`, so might be a better place for such customisation. Thanks for removing, anyway :)